editor/src/clj/editor/boot.clj

```diff
@@ -132,10 +132,11 @@
     (analytics/start! analytics-url analytics-send-interval)
     (Shutdown/addShutdownAction analytics/shutdown!)
     (try
-      (let [game-project-path (get-in opts [:arguments 0])]
+      (let [game-project-path (get-in opts [:arguments 0])
+            game-project-file (io/file game-project-path)]
         (if (and game-project-path
-                 (.exists (io/file game-project-path)))
-          (open-project-with-progress-dialog namespace-loader prefs game-project-path updater false)
+                 (.exists game-project-file))
+          (open-project-with-progress-dialog namespace-loader prefs (.getAbsolutePath game-project-file) updater false)
```

Comment:
We need absolute paths here (that's what `select-project-from-welcome` does, for example). The problem was that the user supplied `game.project` as an argument, and down the line we accessed the project file's parent (i.e. project directory). But, when the path is a single `game.project`, we parent directory of such file is `nil`. So this change fixes the error.